cli/src/main/java/com/liferay/blade/cli/util/BladeUtil.java

```diff
@@ -237,6 +243,39 @@ public static String getManifestProperty(Path pathToJar, String propertyName) th
 		}
 	}
 
+	public static Map<String, ProductInfo> getProductInfo() {
+		try {
+			DownloadCommand downloadCommand = new DownloadCommand();
+
+			downloadCommand.setCacheDir(_workspaceCacheDir);
+			downloadCommand.setPassword(null);
+			downloadCommand.setToken(false);
+			downloadCommand.setUrl(new URL(_PRODUCT_INFO_URL));
+			downloadCommand.setUserName(null);
+
+			//This will be added after LPS-114831 merged.
+			//downloadCommand.setQuiet(true);
```

Comment:
This should be merged now.  can you update the dependency to newest bundle.support and then update this?

Reply:
i use 3.6.0. 3.6.1 seems not published

Reply:
i will send a new pull morning